apps/www/vite.config.ts

```diff
@@ -10,4 +10,7 @@ export default defineConfig({
     },
   },
   plugins: [tsconfigPaths(), envOnlyMacros(), reactRouter()],
+  ssr: {
+    noExternal: ['@navikt/aksel-icons'],
```

Comment:
Should rambda be included as well?

Reply:
Its added as a dependency in the #3758 branch, so figured I update it there.